packages/near-membrane-base/src/__tests__/environment.spec.ts

```diff
@@ -0,0 +1,149 @@
+import { VirtualEnvironment } from '../environment';
+
+describe('VirtualEnvironment', () => {
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
+
+    it('throws when options bag is missing', () => {
+        expect.assertions(1);
+        expect(() => {
+            // @ts-ignore
+            // eslint-disable-next-line no-new
```

Comment:
I wonder why we have a `no-new` lint rule. Is it to use `ReflectConstruct`? 🤷

Reply:
I'm not sure TBH! The rule prevents you from writing `new VirtualEnvironment();` without using the instance for anything, but clearly there is valid cases for doing that 🤷